### PR TITLE
MS2- Using Bootstrap

### LINKED ISSUE #4: responsive columns
Create 3 columns layout responsive for mobile, tablet, and desktop screens 

example
https://cdn.filestackcontent.com/FXC2h6BAQe2m5eNQkyjJ

### PR DESCRIPTION
**Pull request by marijanbrvar**
# The Newsweek Clone
- Setting up content based on archived NW site
- Setting up a responsive layout with flex grid provided by Bootstrap
- Customizing styles for specific elements
Categories covered in this MS are:
Laying out content in tree columns grouped under the following categories:

## Column 1
- Featured stories
- Sponsored insights / Rankings

## Column 2
- Top story
- My turn
- Culture & travel
- More stories
- Latest news

## Column 3
- The debate
- Opinion
- Sponsored insights / Rankings
- Free signup

### CODE REVIEW

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: INVALID! :exclamation:
### Hi @marijanbrvar and @crappypapa   ! :wave: :wave:

#### Hello there, good job completing this project:sunglasses::raised_hands:, however, I have to mark this submission as invalid because you have to re-use the same Pull-Request for the 3 milestones (don't merge after milestone 1 or 2 completion)[Pathwright](https://microverse.pathwright.com/library/new-technical-curriculum/177956/path/step/104328875/).



__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Comment by marijanbrvar** [comment]
> ## STATUS: INVALID! ❗
> ### Hi @marijanbrvar and @crappypapa ! 👋 👋
> #### Hello there, good job completing this project😎🙌; however, I have to mark this submission as invalid because you have to re-use the same Pull-Request for the 3 milestones (don't merge after milestone 1 or 2 completion)[Pathwright](https://microverse.pathwright.com/library/new-technical-curriculum/177956/path/step/104328875/).
> **Kindly make the necessary changes and submit another code review request.**
> _Feel free to reach me on slack for any inquiries about this review._ 😉
> __Grace Tamara Ekunola __
> [GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

Hello Tamara,
We ware merging the previous PR based on an instruction from TSE. See below.
Please advice!

> Project Approved ✔️
> thank you for making the required changes. Your project is now approved!
> 
> please don't forget to merge to master. Good luck with the next projects!
> Best,
> Aziz Mejri
> contact me on Twitter or slack (@Spayco) if you need any help

**Comment by Temmarie** [comment]
Hi @marijanbrvar, it must have been an oversight on his part, you could try to unmerge the branch, I'll advise creating a new branch containing the changes made in that milestone before trying to revert the merge. Check [this](https://stackoverflow.com/questions/28932515/how-to-unmerge-a-git-merge/43292158) link to help revert the merge.

In case reverting doesn't work, leave a note in the pr that you followed the instructions given by the TSE who approved the first milestone overlooking the pathwright instructions, explain the issues you had in reverting the merge and be sure to include a link to the initially approved milestone pr. You could also quote my advice here. 

I hope this helps you have a lovely day!

**Comment by marijanbrvar** [comment]
> Hi @marijanbrvar, it must have been an oversight on his part, you could try to unmerge the branch, I'll advise creating a new branch containing the changes made in that milestone before trying to revert the merge. Check [this](https://stackoverflow.com/questions/28932515/how-to-unmerge-a-git-merge/43292158) link to help revert the merge.
> 
> In case reverting doesn't work, leave a note in the pr that you followed the instructions given by the TSE who approved the first milestone overlooking the pathwright instructions, explain the issues you had in reverting the merge and be sure to include a link to the initially approved milestone pr. You could also quote my advice here.
> 
> I hope this helps you have a lovely day!

Hi,
This is not an issue, I mean unmerge, but the issue is that the Pull request is closed, and you can not open a pull request with the same link, or we can?

Cheers,
Marijan

**Comment by Temmarie** [comment]
@marijanbrvar I'm not sure if this will help, but check [this](https://stackoverflow.com/questions/12674304/github-reopening-a-merged-pull-request) link for reference. Otherwise, as I said before, leave a note in the pr that you followed the instructions given by the TSE who approved the first milestone overlooking the pathwright instructions, explain that you could not re-open the already closed pr, and be sure to include a link to the initially approved milestone pr. You could also quote my advice here.

File: CSS/_article.scss
Comment (teekaytech) [inline CSS/_article.scss]: All commented codes should be completely removed.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__Thank you for informing the next reviewer about the situation of things with your PR. You've done well so far :clap:__

**Top highlights**
- Your Readme looks good :+1: 
- Your PR title and summary is descriptive :100: 
- Your webpage looks organized. Thank you again, for paying attention to the page details. :ok: 


__Although some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] There seems to be an organizing issue in the mobile version of the web page. Check the image below and pay attention to the column title as marked out:

 | Original Page | Cloned page |
 | ------------------ | ----------------- |
 | original | clone |

- [x] For each piece of news, each image should have a link. This is not the case for some news in the `Sponsored Insights` section. Kindly fix this.

- [x] As a requirement, ensure that the sections on your page consecutively follow each other on mobile screen sizes, using the layout below. 
layout


That's it!



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by jubaan — APPROVED** [review]
[![Microverse][microverse-shield]][microverse-url]
<div>
<h1 align="center"> 
  🎉🥳 🎊 🎉 🥳 🥳 🎊 🎉 🥳🥳 🎊 🎉 🥳 🎉 <br />
  🎉🥳 🎊 🎉 🥳 🥳 🎊 🎉 🥳🥳 🎊 🎉 🥳 🎉 <br />
  🥳 🎊 🎉 🥳  APPROVED 🥳 🎊 🎉 🥳 <br />
  🎉🥳 🎊 🎉 🥳 🥳 🎊 🎉 🥳🥳 🎊 🎉 🥳 🎉 <br />
  🎉🥳 🎊 🎉 🥳 🥳 🎊 🎉 🥳🥳 🎊 🎉 🥳 🎉
</h1>
<h2 align="center">
  What’s done is done. What’s gone is gone. <br />
  One of life’s lessons is always moving on. <br />
  It’s okay to look back to see how far you’ve come but <br /> 
  KEEP MOVING FORWARD.
</h2>

<details open="open">
  <summary>Review Summary</summary>
  <ol>
    <li><a href="#optional"> 💡Nice to have and other comments ✍🏼 ✍🏼 </a></li>
    <li><a href="#contact"> 🧔🏼 About your Reviewer </a> (you don't know it... but we are just automated bots 🤖) </li>
    <li><a href="#help"> 🆘 <b> 😭 Feeling stuck? or Need help</b>? 🙏🏼 🙏🏼 </a></li>
  </ol>
</details>
</div>


<h3 id="optional"> 1. 💡 Nice to have and other comments </h3>
<details close="close">
<summary> [click to open] If you have something here, even when approved, it's just a little push to improve your project.</h3></summary>


- [ ] Nothing more to add.


</details>

<h3 id="contact"> 2. 🧔🏼 About your Reviewer </h3>
<details close="close">
<summary> [click to open] Say "hello world!" on Slack: @jubaan</summary>
<br />
<div align="center">
  <a src="https://www.github.com/jubaan">
    Julio Añoveros
  </a>
  <p>
    Julio Añoveros | @jubaan</br></br>
    <b>Full-stack Web Developer</b> and</br>
    🤖 Code Reviewer at <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a>:rocket:
  </p>
</div>
</details>

<h3 id="help"> 3. 🆘 <b> 😭 Feeling stuck? or Need help</b>? 🙏🏼 🙏🏼</h3>
<p> How does it feel? Is nice to advance in the program, right? It keeps you motivated even with all the struggles and takes you one step closer to the goal 🎯!. I wan't you to keep advancing so if you feel stuck and/or need help, remember there is always someone you can reach to keep moving forward, doesn't mater how "dum" you think the questions is.</p>
<details close="close">
  <summary> [click to open] <b>Feel stuck?</b> </summary>
  <br>
  <p> First breath... just do it. Take 15-20 min away from the computer, stretch, meditate, do some exercise, drink water, or do whatever is necessary to distract your mind from the obstacle.</p>
  <p><b>Sometimes you just need to get away to see things with a new perspective.</b></p>
</details>

<details close="close">
  <summary> [click to open] <b>Need help?</b> </summary>
  <br>
  <p> Send me a DM in <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> Students Slack, you'll find me as @jubaan. If I'm not available there are other Code Reviewers you can contact. They'll help as well.</p>
  <p>Remember that <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> has other channels where you can get an answer to your questions:</p>
  <ol>
    <li>The <b>students channel 🧑‍🎓 </b> in slack,</li>
    <li>your <b>teammates 👥 </b>,</li>
    <li>your <b>mentor 👨🏼 👩🏼 </b>, and</li>
    <li><b>Student Success 🏫 </b>.</li>
  </ol>
  <p><b> DON'T hesitate to ask! </b></p>
</details>
<p>If all of the above doesn't help... get a <a href="https://rubberduckdebugging.com/">rubber duck</a> ... it would be great to own one 😄. </p>

<h1 align="center"> See you a the top 🚀!! </h1>

[microverse-shield]: https://img.shields.io/badge/-microverse-%236F23FF?style=for-the-badge
[microverse-url]: https://www.microverse.org/?grsf=kfm2ej
[bootstrap-shield]: https://img.shields.io/badge/Bootstrap-white?style=for-the-badge&logo=bootstrap
[bootstrap-url]: https://www.microverse.org/?grsf=kfm2ej

**Comment by marijanbrvar** [comment]
# 3. Milestone
In the third and last milestone of the project, you will complete the last sections of the website and prepare it for your portfolio.

In the magazine and Editor’s pick
## Newspaper main sections
- US
- World
- Business
- Tech & Science
- Culture
- Sports
- Expert forum
- Subscribe / Newsletter

**Review by smcommits — APPROVED** [review]
# Status: Approved :+1: 
## Hello, team! Great Work! The amount of work put into this project is really impressive. The website looks great. Keep it up! This PR is now approved. You can go ahead and merge the branch. Good Luck!

# 
Any further questions? Please reach  out to me on [slack](https://microverse-students.slack.com/team/U017N7LLZ40)

**Pull request merged**
